database/migrations/2023_10_05_151922_create_stores_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('stores', function (Blueprint $table) {
+            $table->id();
```

Comment:
please replace the cascade on delete
to onDelete('no action')
we don't want the store to be delete if the city is deleted
and to not delete a city that have a store

Reply:
i got it but "not delete a city that have a store"
how can i do it   in migration or we will fix it later in code??

Reply:
@ibrahimAlAssi  instead of onDeleteCascade use onDelete('no action') will do it